content/sigs/advocacy-and-outreach/outreach-programs/index.adoc

```diff
@@ -0,0 +1,85 @@
+---
+layout: project
+title: "Outreach programs"
+section: sigs
+sigId: "advocacy-and-outreach"
+tags:
+  - advocacy-and-outreach
+  - advocacy-and-outreach-sig
+  - outreach-programs
+  - community
+  - outreach
+  - programs
```

Comment:
Increase social medium engagement with a non-default opengraph image that is used for other GSoC pages.
```suggestion
  - programs
opengraph:
  image: /images/gsoc/jenkins-gsoc-logo_small.png
```

Reply:
I have added it. See below.